airflow-core/src/airflow/api_fastapi/common/parameters.py

```diff
@@ -171,43 +171,68 @@ def __init__(
         self.to_replace = to_replace
 
     def to_orm(self, select: Select) -> Select:
+        MAX_SORT_PARAMS = 5
```

Comment:
I think the limit should be made greater.

We already have table with more than 5 columns and the UI could reach that pretty soon.

Going for 10 or 20 shouldn't hurt. (all our different db should be able to handle much more anyway)